### PR TITLE
Setup for Pandas/Numpy additional analysis in End Points

### PR DESCRIPTION
**Pull request by nitish7892**
Adding functionality for possibility to use data frame and pandas library to run statistic analysis. 

Will leverage this to do generate more insights as well as create another end point for the same. 

Added numpy, pandas and django-pandas to the requirements.txt file as well.

### CODE REVIEW

File: backend/requirements.txt [hunk 1]
```diff
@@ -41,3 +41,9 @@ uritemplate==3.0.1
 urllib3==1.25.8
 Werkzeug==1.0.0
 wrapt==1.11.2
+pandas == 1.0.1
+numpy == 1.18.4
+django-pandas== 0.6.1
+pandas == 1.0.1
```
Comment (Skyblueballykid) [inline backend/requirements.txt:47]: You are importing the same packages twice here.

File: backend/requirements.txt [hunk 2]
```diff
@@ -41,3 +41,9 @@ uritemplate==3.0.1
...
+numpy == 1.18.4
+django-pandas== 0.6.1
```
Comment (Skyblueballykid) [inline backend/requirements.txt]: You're importing the same packages twice here.

**Review by Skyblueballykid — CHANGES_REQUESTED** [review]
Looks fine except the same packages are getting imported twice in requirements.txt. Let us know what pandas functions you think you might want to use.

File: backend/requirements.txt (hunk 2 above)
Comment (nitish7892) [reply backend/requirements.txt]: Yes you are right. I will re push it.

**Pull request closed without merging**